CanHazFunny/CanHazFunny.Tests/JesterTests.cs

```diff
@@ -1,10 +1,82 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
 
 namespace CanHazFunny.Tests
 {
     [TestClass]
     public class JesterTests
     {
-        
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void Jester_WithNullJokeService_ThrowsException()
+        {
+            //Arrange
+            Exception? expectedException = null;
+
+            //Act
+            try
+            {
+                Jester jester = new(null, new JokeOutputter());
+            }
+            catch (ArgumentNullException e)
+            {
+                expectedException = e;
+                throw e;
+            }
+
+            //Assert
+            Assert.AreEqual(typeof(ArgumentNullException), expectedException);
+        }
```

Comment:
Excluding this statement on top: [ExpectedException(typeof(ArgumentNullException))] this is the old way of doing it. In class, the professor showed us that we can just use the data row to catch the exception. That would mean we can clean this up significantly: 
```suggestion
        [TestMethod]
        [ExpectedException(typeof(ArgumentNullException))]
        public void Jester_WithNullJokeService_ThrowsException()
        {
                new Jester(null, new JokeOutputter());
        }
```
Because we have the data row it catches the exception for us. If the exception is what we were expecting, the test passes.